readme.txt

```diff
@@ -126,6 +126,7 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 - Update: Adding setup required icon for non-configured payment methods #6811
 - Update: Task list component with new Experimental Task list. #6849
 - Update: Redirect to WC Home after setting up a payment method #6891
+- Dev: Show task and activity notifications in the Inbox panel #7017
```

Comment:
Why `Dev`? Shouldn't this be `Add` or `Update`?

Reply:
You're right, `Add` sounds good to me. This was added in the commit b788750